### ISSUE TITLE
Catch approach rate should not be based on slider velocity

### ISSUE THREAD

**Issue by Xinnoh** [ppy/osu#2606]
The main issue that prevents catch from being reasonably playable is that approach rate is only based on slider velocity. This doesn't work and causes unintended issues.
- Maps with high bpm are unreadable, as the whole map could be ar12, eg. https://osu.ppy.sh/s/511538 
- Maps that use high slider velocities create patterns that are completely unreadable for humans, eg. https://streamable.com/s2ok3

To put it simply, it's like if standard mode had AR only based on slider speed. 
"If speed changes are added, it needs to be a new variable. There is no balanced way to apply a retrospective change to all maps on a larger scale basis"

**Comment by frankhjwx** [comment]
Adding to this, I'm considering if we can use *Inherited* property of a timing point to determine if the timing line is used for speed change for catch mode.

To be more specific, A timing point looks like this for now:
> Offset, Milliseconds per Beat, Meter, Sample Set, Sample Index, Volume, Inherited, Kiai Mode

 While *Inherited* property is only a bool value indicating if it's a red line or a green line, also it's redundant with *Milliseconds per Beat* field as stated in [osu wiki](https://osu.ppy.sh/help/wiki/osu!_File_Formats/Osu_%28file_format%29#timing-points). So I want to suggest adding one more valid value for *Inherited* property, (2 or -1), indicating it's used for catch speed changing things.

Giving an example here:
> 66,315.789473684211,4,2,0,45,1,0  <- means it's a red line with bpm 190
> 10171,-100,4,2,0,60,0,1   <-  means sv 1.0x for a green line
> 10171,-100,4,2,0,60,2,1   <-  means fruit speed 1.0x for a purple line (I'd like it to be purple if realized later)
> 61676,-66.6666666666667,4,2,0,30,0,0   <-  means sv 1.5x for a green line
> 87851,-50,4,2,0,30,0,0 <-  means sv 2.0x for a purple line

We want dev team to know that a normal green sv is commonly used by catch mappers to produce better curves of a **JuiceStream**, and we don't want it to affect the original game play.

**Comment by Xinnoh** [comment]
closed #3427
